tiledb/tests/test_libtiledb.py

```diff
@@ -3477,6 +3477,16 @@ def test_from_numpy_timestamp(self):
         with tiledb.open(path, timestamp=(10, 10)) as A:
             assert A.nonempty_domain() == ((0, 2),)
 
+    @pytest.mark.parametrize("empty_str", ["", b""])
+    @pytest.mark.parametrize("num_strs", [1, 1000])
```

Comment:
Can this start from zero?

Reply:
It is giving this error `TypeError: invalid domain extent, domain cannot be safely cast to dtype dtype('uint64')`. This seems like a different issue, so I can make a new Shortcut story for it.